department-of-veterans-affairs/caseflow
# Queue | Save Notes when navigating back through check out flow

Resolves #5393 

### Description
When navigating away from the `SubmitDecisionView` (i.e. to go back a step) we would reset all decision options (document ID, judge assignment, notes). This preserves those while within a checkout flow

### Acceptance Criteria 
- [ ] When going from Submit Decision back to Select Dispositions and back to Submit Decision, values set in `SubmitDecisionView` are preserved

### Testing Plan
1. Go to Draft Decision checkout flow
2. Set values in `SelectDispositionsView`, go to `SubmitDecisionView`, set values
3. Go back to Select Dispositions, go forwards to Submit Decision. Confirm previously-set values are still set

![submit_decision_opts_preserved](https://user-images.githubusercontent.com/8533004/39761498-423af96a-52a6-11e8-8f0d-e3c8e4d374b7.gif)
